Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -12,9 +12,9 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 2,
-        "Minor": 137,
-        "Patch": 0
+      "Major": 2,
+      "Minor": 137,
```

Comment:
make it 138. They are named based on the sprints unless you plan to port it to 137